tests/runtime/test_device_qir.py

```diff
@@ -0,0 +1,51 @@
+# Copyright (C) 2024 qBraid
+#
+# This file is part of the qBraid-SDK
+#
+# The qBraid-SDK is free software released under the GNU General Public License v3
+# or later. You can redistribute and/or modify it under the terms of the GPL v3.
+# See the LICENSE file in the project root or <https://www.gnu.org/licenses/gpl-3.0.html>.
+#
+# THERE IS NO WARRANTY for the qBraid-SDK, as per Section 15 of the GPL v3.
+
+# pylint: disable=redefined-outer-name,unused-argument,too-many-lines
```

Comment:
Not all of these pylint disable statements are applicable in this shortened / seperated version of the file